src/NexusMods.App.UI/WorkspaceSystem/WorkspaceController/OpenPageBehaviorSettings.cs

```diff
@@ -0,0 +1,24 @@
+using JetBrains.Annotations;
+using MouseButton = Avalonia.Input.MouseButton;
+using KeyboardKey = Avalonia.Input.Key;
+using KeyboardModifiers = Avalonia.Input.KeyModifiers;
+
+namespace NexusMods.App.UI.WorkspaceSystem;
+
+// TODO: Show in Settings (https://github.com/Nexus-Mods/NexusMods.App/issues/396)
+// Related: https://github.com/Nexus-Mods/NexusMods.App/issues/946
+
+[PublicAPI]
+public class OpenPageBehaviorSettings : Dictionary<NavigationInput, OpenPageBehaviorType>
+{
+    public static readonly OpenPageBehaviorSettings Default = new()
+    {
+        // Primary Inputs
+        { new NavigationInput(MouseButton.Left, KeyboardModifiers.None), OpenPageBehaviorType.NewTab },
+        { new NavigationInput(KeyboardKey.Enter, KeyboardModifiers.None), OpenPageBehaviorType.NewTab },
+
+        // Others
+        { new NavigationInput(MouseButton.Left, KeyboardModifiers.Control), OpenPageBehaviorType.ReplaceTab },
+        { new NavigationInput(MouseButton.Left, KeyboardModifiers.Shift), OpenPageBehaviorType.NewTab },
```

Comment:
I'm guessing these are temporary? Should we ask design?
I was thinking that we probably want Middle Click, like in browsers.

Reply:
I forgot the middle click, I'll add that.